yaml-mode.el

```diff
@@ -267,7 +267,7 @@ that key is pressed to begin a block literal."
         (save-excursion
           (forward-char -1)
           (when (and (not (bolp))
-                     (not (memq (preceding-char) '(?\s ?\t))))
+                     (eq ?w (char-syntax (char-before (point)))))
```

Comment:
It's an internal implementation detail that integers (which is what characters are represented as) are `eq` to each other (as they happen to be encoded in pointers, so the pointer equality check `eq` does succeeds). I'd rather use `=` or better, `char-equal` here.

Reply:
Thanks for the comment. Will fix when I get to a computer.